.circleci/config.yml

```diff
@@ -407,7 +407,7 @@ jobs:
       - run:
           name: run shellcheck
           command: |
-            ./s/shellcheck
+            git ls-files --exclude='*.sh' --ignored --cached -z | xargs -0 -r shellcheck
```

Comment:
We need the s/shellcheck script so we can run this check locally

Reply:
Now I have to face that I'm not a developer, and unable to sympathize 😿

Reply:
How about moving this one-liner to `s/shellcheck`?